internal/driver/driver.go

```diff
@@ -270,6 +270,18 @@ func (d *Driver) RemoveDevice(deviceName string, protocols map[string]models.Pro
 	return nil
 }
 
+func (d *Driver) Discover() error {
+	return fmt.Errorf("Driver's Discover function isn't implemented")
```

Comment:
the error message should be in lowercase
https://earthly.dev/blog/golang-errors/

> Lastly, error messages are usually written in lower-case and don’t end in punctuation. Exceptions can be made though, for example when including a proper noun, a function name that begins with a capital letter, etc.

Reply:
Updated.